src/Client.ts

```diff
@@ -50,6 +51,13 @@ export type ListMessagesOptions = {
   limit?: number
 }
 
+export type ListMessagesPaginatedOptions = {
+  startTime?: Date
+  endTime?: Date
+  pageSize?: number
+  direction?: messageApi.SortDirection
```

Comment:
I guess we don't want limit here, because you can just quit the pagination loop whenever you want, correct?

Reply:
That's right